biz.aQute.bnd.exporters/test/biz/aQute/bnd/exporters/subsystem/SubsystemTest.java

```diff
@@ -19,17 +24,28 @@
 import aQute.bnd.osgi.JarResource;
 import aQute.bnd.osgi.Resource;
 import aQute.lib.io.IO;
-import junit.framework.TestCase;
 
-public class SubsystemTest extends TestCase {
-	private String	proj_bundle_1		= "proj_bundle_1";
-	private String	proj_bundle_2		= "proj_bundle_2";
+public class SubsystemTest {
+	private static final String	proj_bundle_1						= "proj_bundle_1";
+	private static final String	proj_bundle_2						= "proj_bundle_2";
+
+	private static final String	proj_bundle_1jar					= proj_bundle_1 + "-0.0.1.jar";
+	private static final String	proj_bundle_2jar					= proj_bundle_2 + "-0.0.1.jar";
+
+	private static final String	RESULTS_PATH						= "results/";
+	private static final String	WS_PATH								= "test-ws/";
+	private static final String	GEN_WS_PATH							= "generated/" + WS_PATH;
```

Comment:
Could we please put it in a different folder per test case `"generated/tmp/test/"+testMethodName`? (You can get the test method name with a rule:

```java
	@Rule
	public TestName					testName	= new TestName();
```
)

Using a different folder per test method avoids side effects and potential for windows file locking issues.